creator-node/src/config.js

```diff
@@ -285,7 +285,7 @@ const config = convict({
     doc: 'Whether or not cluster logic should be enabled (running multiple instances of the app to better utuilize multiple logical cores)',
     format: Boolean,
     env: 'clusterModeEnabled',
-    default: true
+    default: false
```

Comment:
woah why change this - this is the prod setting

Reply:
oh yeah should flag that we'd need to make sure a docker-compose PR gets merged before this one to make sure it's enabled. this is just a convenience change for local dev because it gets annoying for debugging. also would have to consider how to make it work for the provider who doesn't use docker-compose, but I'm not as familiar with that

if it's too much work just revert the config change for now. ideally we could have the default be disabled for local and enabled for maddog+stage+prod

Reply:
Was changing it based on this - https://github.com/AudiusProject/audius-protocol/pull/4367#pullrequestreview-1183435767